connector/ldap/ldap.go

```diff
@@ -68,7 +68,10 @@ type Config struct {
 
 	// Path to a trusted root certificate file.
 	RootCA string `json:"rootCA"`
-
+	// Path to a self-signed certificates ca file.
+	ClientCA string `json:"clientCA"`
```

Comment:
❓ Wouldn't this mostly match the `RootCA`? I'm not sure I understand the usecase where it differs. Can you please elaborate?

Reply:
If ldap server setting verify client cert is true, only setting RootCA cannot work.